acs5countypoverty_child.py

```diff
@@ -0,0 +1,54 @@
+from census_response import getCensusResponse
+import json
+
+def main():
+    #Estimate!!Total!!Population for whom poverty status is determined
+    pop_table = 'S1701_C01_001E'
+    #Estimate!!Below poverty level!!Population for whom poverty status is determined!!AGE!!Under 18 years
+    child_poverty_table = 'S1701_C02_002E'
+    
+    #If additional subdivision are needed
+    #'S1701_C02_003E' = AGE!!Under 18 years!! Under 5 years!!
+    #'S1701_C02_004E' = AGE!!Under 18 years!! 5 to 17 years!!
+
+    subject_table = 'https://api.census.gov/data/2018/acs/acs5/subject?'
+    get_ls = [pop_table, child_poverty_table]
+    #county level request for IL (state 17)
+    geo = 'county:*&in=state:17'
+
+    response = getCensusResponse(subject_table, get_ls, geo)
+
+    #poverty_data is list of lists, where list elements are rows
+    #header row: name, population total, population poverty, fip state, fip county
+    poverty_data = response.json()
+
+    #Json format
+    #{'countyfip': (countyfip = concatenation state and county fip numbers)
+        #{'metric_one': 1234,
+        #'metric_two': 5678}}
+        
+    final_json = {}
+
+    for d in poverty_data[1:]:
+        #set variables to list elements
+        name, pop_total_str, pop_poverty_str, fip_state, fip_county = d
+        #convert strings to ints
+        pop_total_int = int(pop_total_str)
+        pop_poverty_int = int(pop_poverty_str)
+        #concat strings
+        fip_final = fip_state + fip_county
+        #calculate percent poverty
+        pct_poverty = pop_poverty_int / pop_total_int * 100
+        #create county json
+        county_json = {'name_county':name, 'population_total': pop_total_int, 'population_poverty': pop_poverty_int, 'percent_poverty': pct_poverty}
```

Comment:
Because the population_total is from "Estimate!!Total!!Population for whom poverty status is determined" and I am not sure if that is the total population. We should group these metrics like we group the race one:
`{'name_county':name, 'poverty_metrics':{...}}"`

And we want to nest another dictionary for percentages.